client/my-sites/stats/stats-module/index.jsx

```diff
@@ -217,12 +224,14 @@ export default connect( ( state, ownProps ) => {
 	const siteSlug = getSiteSlug( state, siteId );
 	const { statType, query } = ownProps;
 	const gateStats = shouldGateStats( state, siteId, statType );
+	const gateDownloads = shouldGateStats( state, siteId, 'download-csv' );
```

Comment:
I'm thinking one of the goals is to prevent hardcoded strings for stats in upcoming PRs. And we have a single file that defines the statTypes and features.